harness/terraform-provider-harness
# fix: [CCDS-101356]: Prevent users to configure multiple blocks for gitops resources where this is applicable

**Title:** Prevent users to configure multiple blocks for gitops resources where this is applicable

**Summary:**
Add max items (1) validation for multiple GitOps resources. User shouldn't be able more than one blocks for these resources as it wouldn't make sense. 

**Testing Instructions:**
Configure these resources with single or multiple blocks

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`
- gitleaks: `trigger gitleaks`
